### PR TITLE
[PLAYER-4099],[PLAYER-4101] 

### PR DESCRIPTION
**Pull request by cacel**
Adding a textView to retrieve data from analytics offline

### CODE REVIEW

File: PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -246,6 +247,7 @@
 		CAC620481D63C9AB004902F3 /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				B8066C4A215B932200E573D0 /* OoyalaSDK.framework */,
```
Comment (maxkupetskii) [inline PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp.xcodeproj/project.pbxproj]: Are these changes necessary?

File: PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -95,7 +96,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				CAFA5BAE1DDD400500CAE010 /* OoyalaSDK.framework in Frameworks */,
+				B8066C4B215B932200E573D0 /* OoyalaSDK.framework in Frameworks */,
```
Comment (maxkupetskii) [inline PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp.xcodeproj/project.pbxproj]: Are these changes necessary?

File: PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		B8066C4B215B932200E573D0 /* OoyalaSDK.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B8066C4A215B932200E573D0 /* OoyalaSDK.framework */; };
```
Comment (maxkupetskii) [inline PlaybackLab/DownloadToOwnSampleApp/DownloadToOwnSampleApp.xcodeproj/project.pbxproj]: Are these changes necessary?

**Review by brol1dev — APPROVED** [review]
apart from @maxkupetskii comments, LGTM

**Comment by cacel** [comment]
I remove the pbxproj from the PR. @rcvrgs @maxkupetskii

**Review by ipadilla4 — APPROVED** [review]
LGTM
